code-studio/src/js/initApp/project.js

```diff
@@ -129,6 +129,13 @@ var projects = module.exports = {
   },
 
   /**
+   * Whether this project uses Firebase for data storage.
+   */
+  getUseFirebase: function () {
+    return current.useFirebase;
+  },
```

Comment:
Done. I think I was trying to avoid a name collision without realizing these were hanging off different objects.